src/components/Pad.js

```diff
@@ -34,5 +43,7 @@ const PadStyled = styled.div`
     top: -0.7rem;
     font-size: 0.5rem;
     color: #3d4448;
+    cursor: default;
```

Comment:
Ist a bit strange, that the span-tag has curser:pointer and the pads default. Maybe here is better pointer as well?